src/main/webapp/script.js

```diff
@@ -39,6 +46,30 @@ function loadSignup() {
   fetchMentorExperience();
 }
 
+/**
+ * Fetches a single question and its answers from server, 
+ * wraps each in an <li> element, and adds them to the DOM.
+ */
+async function fetchAnswers() {
```

Comment:
It might be more clear to change this comment to say "Fetches answers for a single question from the server..." because it doesn't look like this method is actually fetching the question itself.

Reply:
yeah, leftover comment from past ideas, done